CSS/Comments.tmPreferences

```diff
@@ -14,13 +14,13 @@
 				<key>name</key>
 				<string>TM_COMMENT_START</string>
 				<key>value</key>
-				<string>/*</string>
+				<string>/* </string>
```

Comment:
I'm not interested in changing the whitespace on any comment markers at this point. There doesn't seem to be any functional benefit, just personal preference.